content/docs/ui/managing-contacts/email-address-validation.md

```diff
@@ -84,15 +85,25 @@ The API response should look something like this, using the example email "cedri
 "has_suspected_bounces":false}},"ip_address":"65.101.243.157"}}
 ```
 
-## Integration suggestions
+### Options for Integration
 
-While the API allows flexibility for integrating in multiple ways, this section describes a recommended approach for validating emails in a sign up form. This could be a sign up form for subscribing to a newsletter, or for signing up for an account with your service. Some websites require visitors to enter their email address in twice, but with Twilio SendGrid’s Email Validation API, you can require that they only enter it in *once*, reducing friction points while also validating user input.
+The Email Validation API allows flexibility for integrating in multiple ways, but for simplicity we'll focus on validating emails in a signup form. A signup form is anywhere you are capturing email addresses. This could be a lead capture signup form like requesting a content piece or an account signup. 
 
-### When to call the API
+Once you identify the signup form you want to integrate with the Email Validaiton API, next you need to decide if you want to use validation results to surface possible typos to form visitors while they're still in the form or capture the results after the form submission to inform sending decisions. 
 
-Call the API in your backend code, most commonly after the user submits the form. You could also call your backend to validate an email address right after a user types in their email, but the most important thing is that the API call is made in your backend code, thereby protecting your API key from prying eyes.
+The first option is to provide user feedback by returning to the sign up form and showing an error screen, letting the user know that they need to check the email address. For instance, you might realize the user made a typo as described in the previous section and decide to show an error like, “Did you mean john.doe@gmail.com?”
+
+<call-out type="warning">
+
+Be careful not to create a block in your sign up form if you provide user feedback. It’s possible that a user’s actual email address doesn’t meet your criteria for a good address, and will never be able to successfully submit your form. You should consider alerting the user only once, and then allowing the same address if they submit it again a second time.
 
-### Evaluating an email address verdict
```

Comment:
Is this where the table would go to replace the explanations of verdict/score/checks?